tests/ee_searchable_tokenization_config.yaml

```diff
@@ -26,6 +26,33 @@ schemas:
         token_type: email
         consistent_tokenization: true
 
+  - table: test_tokenization_default_client_id_join
+    columns:
+      - id
+      - nullable
+      - empty
+      - token_i32
+      - token_i64
+      - token_str
+      - token_bytes
+      - token_email
+    encrypted:
+      - column: token_i32
+        token_type: int32
+        consistent_tokenization: true
```

Comment:
idea for the future: support specifying `consistent_tokenization` in the `default` section once and apply by default )) can you create task for that, please?)

Reply:
Done `T2674` created